app/code/Magento/Catalog/Model/CategoryLinkRepository.php

```diff
@@ -87,6 +101,7 @@ public function deleteByIds($categoryId, $sku)
         try {
             $category->save();
         } catch (\Exception $e) {
+            $this->logger->critical($e->getMessage());
```

Comment:
Don't we have a separate method to log exception?

Reply:
@orlangur seems like we don't.

Reply:
Ok, will try to grep later.